### DISCUSSION TITLE
Could not find com.github.afreakyelf:Pdf-Viewer:Tag.

### DISCUSSION THREAD

**Discussion by siddhant792** [afreakyelf/Pdf-Viewer#15] [Q&A] [answered]
PdfViewerActivity could not be resolved in my project!! Can anyone please help me
![Capture](https://user-images.githubusercontent.com/44133518/113397046-132f5000-93ba-11eb-8044-3b1eafee251e.PNG)

**Comment by afreakyelf** [comment]
Hi, Tag refers to latest version. For the latest version refer to readme file Jitpack badge.
Currently it's 1.0.7

Also if you are using Java, please use _PdfViewerActivity.companion.buildntent....._

Thank you

**Reply by sagarbandamwar** [reply]
do we need to add "v" before 1.0.7 ?
for me its not loading

**Reply by afreakyelf** [reply, marked as answer]
Yes you have to.
